src/main/webapp/src/app/ranking/ranking-list/ranking-list.component.spec.ts

```diff
@@ -0,0 +1,26 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { HttpClientModule } from '@angular/common/http';
+import { HttpClientTestingModule, HttpTestingController } from '@angular/common/http/testing';
+
+import { RankingListComponent } from './ranking-list.component';
+import { HiddenGemService } from '../../hidden-gem.service';
+
+
+describe('HiddenGemService', () => {
```

Comment:
Why is there a hidden gems service test in the ranking list component spec?

Reply:
Oh yes sorry this is wrong, I thought I pushed the new test I have written for ranking list but I did not... The ranking list component test is now pushed, let me know if there is any more issues.